hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/SingleTransactionRecordBuilderImpl.java

```diff
@@ -243,7 +251,10 @@ public SingleTransactionRecord build() {
         final Timestamp consensusTimestamp = HapiUtils.asTimestamp(consensusNow);
         final Timestamp parentConsensusTimestamp =
                 parentConsensus != null ? HapiUtils.asTimestamp(parentConsensus) : null;
-
+        // sort the automatic associations to match the order of mono-service records
+        automaticTokenAssociations.sort(
```

Comment:
I'd lean toward making defining this `Comparator` in a `private static final` field.

Reply:
Makes sense. Done. Thanks